src/core/chainOperator/chainAdapters/cosmjs.ts

```diff
@@ -155,6 +174,53 @@ class CosmjsAdapter implements ChainOperatorInterface {
 		return mempoolResult.result;
 	}
 
+	/**
+	 * Sets new Clients for Mempoolloop.
+	 *
+	 */
+	public async getNewClients(): Promise<string | void> {
```

Comment:
So is this function different between cosmjs and injective?

Reply:
so we should add this to the chainoperator and only change implementation between the different adapters